src/auto_version/tests/example2.py

```diff
@@ -0,0 +1,6 @@
+LOCK = False
+RELEASE = True
+VERSION = "19.99.0"
```

Comment:
Literally the same file as `example.py` - we just need to hit at least one variable that was already substituted elsewhere.